extensions/amp-link-rewriter/0.1/test/validator-amp-link-rewriter.html

```diff
@@ -0,0 +1,47 @@
+<!--
+  Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS-IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the license.
+-->
+<!--
+  Test Description:
+  Valid amp-link-rewriter tag
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="./regular-html-version.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async custom-element='amp-link-rewriter' src='https://cdn.ampproject.org/v0/amp-link-rewriter-0.1.js'></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+<amp-link-rewriter layout="nodisplay">
+  <script type="application/json">
+    {
+      "output": "https://visit.digidip.net?pid=110&url=${href}&cid=${data.customerId}&mid=${data.merchantId}&ref=${referrer}&location=${location}&rel=${rel}",
+      "section": [
+        "#track-section"
+      ],
+      "attribute": {
+        "href": "^((?!\bgmail\\.com\b|\byoutube\\.com\b).)*$",
+        "class": "content"
+      },
+      "vars": {
+        "customerId": "12345",
+        "merchantId": "3234"
+      }
+    }
+  </script>
+</amp-link-rewriter>
```

Comment:
Include an invalid version. An invalid one missing the child script tag would be

```
<amp-link-rewriter layout="nodisplay"></amp-link-rewriter>
```